gatsby-browser.js

```diff
@@ -1 +1,19 @@
 export {wrapPageElement} from './gatsby-ssr';
+
+// // gatsby-browser.js
+export const onRouteUpdate = ({location}) => {
+  // Client-side redirects for links with anchors
+  if (location.hash.length > 0) {
+    const path = location.pathname + location.hash;
+    if (Object.keys(redirects).includes(path)) {
+      const redirectURL = location.origin + redirects[path];
```

Comment:
We should copy over the search too, just in case there is any. ex. `?urm_param=foo`
```suggestion
      const redirectURL = location.origin + redirects[path] + location.search;
```